app/src/main/java/org/mozilla/fenix/settings/quicksettings/QuickSettingsSheetDialogFragment.kt

```diff
@@ -102,12 +104,19 @@ class QuickSettingsSheetDialogFragment : AppCompatDialogFragment() {
     }
 
     override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
-        val customDialog = if (promptGravity == BOTTOM) {
-            return BottomSheetDialog(requireContext(), this.theme)
+        return if (promptGravity == BOTTOM) {
+            val bottomSheetDialog = BottomSheetDialog(requireContext(), this.theme)
+            bottomSheetDialog.setOnShowListener {
+                val bottomSheet = bottomSheetDialog.findViewById<View>(
+                    com.google.android.material.R.id.design_bottom_sheet
+                ) as? FrameLayout
+                val behavior = BottomSheetBehavior.from(bottomSheet)
+                behavior.state = BottomSheetBehavior.STATE_EXPANDED
+            }
+            bottomSheetDialog
```

Comment:
nit: Use `.apply` instead of a local variable for the bottom sheet.

Reply:
Thanks!